examples/projects/bsoncxx/cmake-deprecated/shared/build.sh

```diff
@@ -10,13 +10,12 @@ rm -rf build/*
 cd build
 if [ -z "$MSVC" ]; then
     "$CMAKE" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" -DCMAKE_CXX_STANDARD="${CXX_STANDARD}" ..
-    make run VERBOSE=1
```

Comment:
Replace with `make run` to run example. Same comment applies to other files under `examples/projects`.